browser/go/conversion_test.go

```diff
@@ -0,0 +1,107 @@
+// +build js,wasm
+
+package main
+
+import (
+	"encoding/json"
+	"syscall/js"
+	"testing"
+	"time"
+
+	"github.com/0xProject/0x-mesh/core"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConvertConfigEmpty(t *testing.T) {
+	data, err := json.Marshal(struct {
```

Comment:
This type of JSIN marshaling makes me a lot more comfortable. I'm not sure it's 100% needed, but at least it's a "dumb" conversion and doesn't go through any of our other methods that might do additional conversion.